Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build node feature discovery
-FROM golang:1.8 as builder
+FROM BASEIMAGE1 as builder
```

Comment:
How about getting rid of `sed` magic and using build args, instead? I.e. something like:
```
ARG  BASEIMAGE1_ARCH
FROM ${BASEIMAGE1_ARCH}golang:1.11-stretch
```
Then, the Dockerfile could be used directly for the basic use case, without any mangling.

Reply:
Done